zircon.core/src/commonMain/kotlin/org/hexworks/zircon/internal/data/DefaultStackedTile.kt

```diff
@@ -78,4 +67,24 @@ class DefaultStackedTile(
     override fun asImageTile(): Maybe<ImageTile> = Maybe.empty()
 
     override fun asGraphicTile(): Maybe<GraphicalTile> = Maybe.empty()
+
```

Comment:
All 3 methods (`toString`, `equals` and `hashCode`) can be replaced by using a `data class` instead of `class`. More on `data class`es [here](https://kotlinlang.org/docs/reference/data-classes.html). 👍